src/frontend/web_application/src/layouts/Settings/presenter.jsx

```diff
@@ -1,32 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import NavList, { ItemContent } from '../../components/NavList';
 import MenuBar from '../../components/MenuBar';
-import Link from '../../components/Link';
-
-import './style.scss';
 
 const Settings = ({ __, children }) => {
```

Comment:
may be we can connect this component to detect active link and so we keep MenuBar as dumb as possible